quinn-udp/Cargo.toml

```diff
@@ -22,6 +22,7 @@ log = ["tracing/log"]
 maintenance = { status = "experimental" }
 
 [dependencies]
+bytes = "1"
```

Comment:
Would you mind squashing this change into the first commit? We prefer to rebase merge clean commits into main rather than squashing.

Reply:
I was assuming you are squashing it entirely, will do

Reply:
should be good now